### PR TITLE
PR @eng-dibo

### PR DESCRIPTION
**Pull request by eng-dibo**
- export class App in the generated server/server.ts,
 so external services can import it, such as firebase-functions.

### CODE REVIEW

File: src/lib/generators/server/generate-server.ts [hunk 1]
```diff
@@ -402,13 +403,13 @@ export const generateServer = (ui: UI): OptionalKind<SourceFileStructure> => {
...
-              "this.express.use(`/${process.env.API_NAME}/media-objects`, new MediaObjectRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/endpoints`, new EndpointRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/books`, new BookRoute().router);",
+              "this.express.use(`/${this.api}/auth`, new AuthRoute().router);",
+              "this.express.use(`/${this.api}/users`, new UserRoute().router);",
+              "this.express.use(`/${this.api}/resource-permissions`, new ResourcePermissionRoute().router);",
+              "this.express.use(`/${this.api}/roles`, new RoleRoute().router);",
+              "this.express.use(`/${this.api}/media-objects`, new MediaObjectRoute().router);",
+              "this.express.use(`/${this.api}/endpoints`, new EndpointRoute().router);",
+              "this.express.use(`/${this.api}/books`, new BookRoute().router);",
```
Comment (Netherium) [inline src/lib/generators/server/generate-server.ts:412]: This is related to .env
You can take a look in .env.sample for more info.

File: src/lib/generators/server/generate-server.ts [hunk 2]
```diff
@@ -1,338 +1,367 @@
-import { ImportDeclarationStructure, OptionalKind, Scope, SourceFileStructure, StructureKind, VariableDeclarationKind } from 'ts-morph';
-import { UI } from '../../interfaces/ui.model';
-import pluralize from 'pluralize';
```
Comment (Netherium) [inline src/lib/generators/server/generate-server.ts:3]: These settings are related to .editorconfig
Engage it or use a better IDE like WebStorm/PhpStorm
False positive #3

File: src/lib/generators/server/generate-server.ts (hunk 1 above)
Comment (eng-dibo) [reply src/lib/generators/server/generate-server.ts:412]: `this.api` is equivalent to `${process.env.API_NAME}` and already defined in the constructor.

File: src/lib/generators/server/generate-server.ts (hunk 2 above)
Comment (eng-dibo) [reply src/lib/generators/server/generate-server.ts:3]: I already use visual studio code, but there is no linter guide added to your repo, we eill add one later, so all contributors have the same code style.

Comment (Netherium) [reply src/lib/generators/server/generate-server.ts:3]: Use .editorconfig.
False positive again for 6th time.

**Comment by Netherium** [comment]
App class is already exported.
It could not be tested otherwise.

**Pull request closed without merging**